pkg/test/utils/kubernetes.go

```diff
@@ -952,23 +953,34 @@ func RunCommand(ctx context.Context, namespace string, command string, cmd harne
 		fmt.Sprintf("PATH=%s/bin/:%s", actualDir, os.Getenv("PATH")),
 	}
 
-	err = builtCmd.Run()
+	err = builtCmd.Start()
 	if err != nil {
 		if _, ok := err.(*exec.ExitError); ok && cmd.IgnoreFailure {
-			return nil
+			return nil, nil
 		}
+		return nil, err
 	}
 
-	return err
+	if cmd.Background {
```

Comment:
@kensipe do you know if we want to use the same `stdout` and `stderr` as test commands for the background processes?  
Basically using any redirect pipe `>` would be enough to redirect bg process logs to where user wants and not mix them up with test logs?

Reply:
ah... no and I thought I handled that... thanks..  let me check on that

Reply:
Yeah, intermingled logs are no help.

Reply:
totally agree!  It was a mistake for sure... thanks for the eagle eyes